docs/users/index.md

```diff
@@ -37,8 +37,8 @@ It is a complete health information system allowing users to customize it by cho
 |ERP|Odoo|`14`|:material-diamond-stone: Core|
 |ERP|:erpnext: ERPNext|`15.12.2`|:material-timer-pause-outline: Legacy|
 |BI|:superset: Apache Superset|`3.1.0`|:material-diamond-stone: Core|
-|IdP|:keycloak: JBos Keycloak|`22.0.5`|:material-diamond-stone: Core|
-|PACS|Orthanc|`1.0.0`|:material-diamond-stone: Core|
+|IdP|:keycloak: JBoss Keycloak|`22.0.5`|:material-diamond-stone: Core|
+|PACS|Orthanc|`1.12.6`|:material-diamond-stone: Core|
```

Comment:
@kdaud did you verify that **all** the versions are correct as per `alpha.13`?

Reply:
> did you verify that all the versions are correct as per alpha.13?

Yes.